src/config.rs

```diff
@@ -1,10 +1,11 @@
-use std::{ffi::OsString, path::PathBuf};
+use std::{ffi::OsString, path::PathBuf, sync::Arc};
 
 #[doc(hidden)]
 pub struct Config {
     pub(crate) arguments: Vec<OsString>,
     pub(crate) log_command: bool,
     pub(crate) working_directory: Option<PathBuf>,
+    pub(crate) stdin: Arc<Vec<String>>,
```

Comment:
After #67 I had to either pass in a `&mut Config` into `Waiter::spawn_standard_stream_relaying` (which I think is weird) or make this an `Arc`. I think this is fine. cc:  @casey

Reply:
I think having an `Arc` here is actually better than the solution I had before merging in #67. In that state I was mutating the passed in `Config` to avoid cloning of `stdin`.

Reply:
I think the `Arc` is good.

Reply:
You could also use `std::mem::swap` to move out of the config. It is a little weird to mutate the config though.

Reply:
Yeah, that's what I had before #67. But I think it's better to have an `Arc` than to have mutability.